tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -161,7 +161,7 @@ def tap_update(self, return_details=True):
             return ContactDetails(self.marionette)
         else:
             # else we drop back to the underlying app
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != self.name)
+            Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url != self.manifest_url)
```

Comment:
but the static method used is when the displayed_app.manifest_url is equal "==" to self.manifest_url https://github.com/mermi/gaia/commit/7e43e5457a8d60f3d0c0bf8d525e5930bf8f7e66#diff-06f57e3ac9df83884ad662cda7599c3bR159